spot-ml/src/main/scala/org/apache/spot/SuspiciousConnects.scala

```diff
@@ -103,7 +103,11 @@ object SuspiciousConnects {
             InvalidDataHandler.showAndSaveInvalidRecords(invalidRecords, config.hdfsScoredConnect, logger)
           }
 
-          case None => logger.error("Unsupported (or misspelled) analysis: " + analysis)
+          case None => logger.error(s"Something went wrong while trying to run Suspicious Connects Analysis")
+            logger.error(s"The value of parameter analysis (provided: $analysis) is any of the valid analysis types? " +
```

Comment:
Suggested change?: "Is the value of the analysis parameter  (provided: $analysis) any of the valid analysis types?"